kalite/topic_tools/annotate.py

```diff
@@ -95,13 +91,6 @@ def update_content_availability(content_list, language="en", channel="khan"):
                         "stream_type": "{kind}/{format}".format(kind=content.get("kind").lower(), format=format),
                         "thumbnail": thumbnail,
                     }
-            elif django_settings.BACKUP_VIDEO_SOURCE:
```

Comment:
Actually there's an error in this block of code. See my commit message:

```
That setting is referenced nowhere else, and furthermore the block
of code uses an undefined name "dubbed_id". So if executed, it would
result in an error.

It's not obvious to me what value *should* go there, hence removing it.
```